k8s/upgrades/0.9.0-1.0.0/cstor/cstor_pool_upgrade.sh

```diff
@@ -27,41 +29,25 @@ function usage() {
     exit 1
 }
 
-##Checking the version of OpenEBS ####
-function verify_openebs_version() {
-    local resource=$1
-    local name_res=$2
-    local openebs_version=$(kubectl get $resource $name_res -n $ns \
-                 -o jsonpath="{.metadata.labels.openebs\.io/version}")
+function pre_check() {
+    local ns=$1
+    local pod_version=""
+    ## name=maya-apiserver label is common for both helm and operator yaml
+    maya_pod_name=$(kubectl get pod -n $ns \
+                 -l name=maya-apiserver    \
```

Comment:
when installed with helm, this might be called `openebs-apiserver`

Reply:
Its constant label  `name=maya-apiserver ` @kmova it will not  change.